projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -29,6 +29,11 @@ export interface FeatureTogglesInterface {
    */
   formErrorsDescriptiveMessages?: boolean;
 
+  /**
+   * After the feature toggle is enabled, Customer Support Agent logs in ASM with the Authorization Code flow.
+   */
+  showOauth2AsmloginPage?: boolean;
```

Comment:
Please note that with feature flags, the flag will be enabled in newly-created Spartacus apps.  This can interfere with E2E testing until a JDK21 backend is used regularly.

Reply:
Expanding on this warning, the current code is fine if you are planning to wait until August to merge this PR for the september release.  

If you wish to merge is sooner, you will need to do 2 things:
1. Convert the feature flag to a "feature provider" (we have no formal term for this construct), like we did with `provideAuthorizationCodeFlowByDefault()` in develop.
2. Create a PR to convert the "feature provider" back into a feature toggle, with the plan to merge this PR in August so that it is included with the September release.